### ISSUE TITLE
How to place footnotes at the bottom of the page?

### ISSUE THREAD

**Issue by Ziriax** [Mogztter/asciidoctor-web-pdf#309]
I've read that this is impossible. 

From a developer point of view: I guess that's because one would need multiple render/layout passes to figure out how much content to place on a page so that room is available at the bottom for the footnotes on that page? And when content is removed that contained footnotes, this becomes a really hard problem?

What solutions could be suggested for this? I want readers of my book to be able to read the footnotes on page N on that same page...

**Comment by Mogztter** [comment]
Hello @Ziriax 

The [Paged.js](https://www.pagedjs.org/) team is working on this feature so I think the best solution is to help them implement this feature:

- https://mattermost.pagedmedia.org/pagedmedia/channels/town-square
- https://gitlab.pagedmedia.org/tools/pagedjs

Since Paged.js will paginate the content and do all the computation, I think this is the only way.

**Comment by larshp** [comment]
relevant pagedjs issue: https://gitlab.pagedmedia.org/tools/pagedjs/issues/161

**Comment by danyill** [comment]
Excitingly :tada: [a new release of pagedjs was made yesterday](https://gitlab.pagedmedia.org/tools/pagedjs/-/tags/v0.2.0) with support for this!

**Comment by zeFresk** [comment]
I got footnotes working by manually updating `paged.js` to 0.2.0 in package.json and then editing a line of `lib/document/document-coverter.js`.

Then I added this in my .css (I don't know if it was necessary)

```css
@page {
	@footnote {
		float: bottom;
		border-top: solid black thin;
		padding-top: 8pt;
	}
}

span.footnote {
	float: footnote;
}
```

Finally, I replaced all my asciidoctor footnotes `footnote:[bla]` with a `<span class="footnotes">` like that:

```adoc
[.footnote]#bla#
```

It works but fails one test of `npm run test`.

I attached a patch.

[patch.txt](https://github.com/Mogztter/asciidoctor-web-pdf/files/6968982/patch.txt)
